astropy/io/ascii/latex.py

```diff
@@ -310,16 +276,14 @@ def __init__(self, ignore_latex_commands=['hline', 'vspace', 'tableline'],
         self.data.comment = self.header.comment
 
     def write(self, table=None):
-        self.header.start_line = None
-        self.data.start_line = None
+        self.header.start_line = 0
+        self.data.start_line = 0
```

Comment:
I don't think these should be changed.  `None` has a particular meaning here, which is to take the default (which may end up still being `0`, but it could break other expectations if the default is not indicated as `None`).